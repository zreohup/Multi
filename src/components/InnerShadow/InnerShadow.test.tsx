import { render } from '@/src/tests/test-utils'
import { InnerShadow } from '.'

describe('InnerShadow', () => {
  it('should render the default markup', () => {
    const container = render(<InnerShadow />)

    expect(container).toMatchSnapshot()
  })
})
