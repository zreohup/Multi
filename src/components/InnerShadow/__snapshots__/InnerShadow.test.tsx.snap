// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InnerShadow should render the default markup 1`] = `
<View>
  <View
    style={
      {
        "backgroundColor": "#FFFFFF",
        "bottom": 0,
        "height": 10,
        "left": 0,
        "position": "absolute",
        "shadowColor": "rgb(255,255,255)",
        "shadowOffset": {
          "height": -4,
          "width": -2,
        },
        "shadowOpacity": 1,
        "shadowRadius": 4,
        "width": "100%",
      }
    }
  />
</View>
`;
